xsd/NeTEx_publication.xsd

```diff
@@ -6365,30 +6525,18 @@ Provides a general purpose wrapper for NeTEx data content.</xsd:documentation>
 			<xsd:field xpath="@version"/>
 		</xsd:keyref>
 		<xsd:key name="BorderPoint_AnyVersionedKey">
-			<xsd:selector xpath=".//netex:BorderPoint  "/>
+			<xsd:selector xpath=".//netex:BorderPoint"/>
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
 		<!-- =====FareProduct============================== -->
-		<!-- =====FareProduct Key ========================== -->
```

Comment:
Where did this go to?

Reply:
This should be looked into, as it seems the refactoring at least have missed this concept in the NeTEx_publication_timetable.xsd
See output from 
  grep -irHn5 "FareProduct Key" *
for v1.09 and v1.1 (May 17th 2019).

Reply:
(file attachement failed)

**v1.0:**

xsd/NeTEx_publication.xsd-6368-			<xsd:selector xpath=".//netex:BorderPoint  "/>
xsd/NeTEx_publication.xsd-6369-			<xsd:field xpath="@id"/>
xsd/NeTEx_publication.xsd-6370-			<xsd:field xpath="@version"/>
xsd/NeTEx_publication.xsd-6371-		</xsd:key>
xsd/NeTEx_publication.xsd-6372-		<!-- =====FareProduct============================== -->
xsd/NeTEx_publication.xsd:6373:		<!-- =====FareProduct Key ========================== -->
xsd/NeTEx_publication.xsd-6374-		<!-- 
xsd/NeTEx_publication.xsd-6375-		<xsd:keyref name="FareProduct_KeyRef" refer="netex:FareProduct_AnyVersionedKey">
xsd/NeTEx_publication.xsd-6376-			<xsd:selector xpath=".//netex:FareProductRef |.//netex:SupplementToFareProductRef "/>
xsd/NeTEx_publication.xsd-6377-			<xsd:field xpath="@ref"/>
xsd/NeTEx_publication.xsd-6378-			<xsd:field xpath="@version"/>
--
--
xsd/NeTEx_publication.xsd-6474-			</xsd:annotation>
xsd/NeTEx_publication.xsd-6475-			<xsd:selector xpath=".//netex:PreassignedFareProduct  "/>
xsd/NeTEx_publication.xsd-6476-			<xsd:field xpath="@id"/>
xsd/NeTEx_publication.xsd-6477-			<xsd:field xpath="@version"/>
xsd/NeTEx_publication.xsd-6478-		</xsd:unique>
xsd/NeTEx_publication.xsd:6479:		<!-- =====PreassignedFareProduct Key ========================== -->
xsd/NeTEx_publication.xsd-6480-		<xsd:keyref name="PreassignedFareProduct_KeyRef" refer="netex:PreassignedFareProduct_AnyVersionedKey">
xsd/NeTEx_publication.xsd-6481-			<xsd:selector xpath=".//netex:PreassignedFareProductRef"/>
xsd/NeTEx_publication.xsd-6482-			<xsd:field xpath="@ref"/>
xsd/NeTEx_publication.xsd-6483-			<xsd:field xpath="@version"/>
xsd/NeTEx_publication.xsd-6484-		</xsd:keyref>
--
--
xsd/NeTEx_publication_timetable.xsd-6382-			<xsd:selector xpath=".//netex:BorderPoint  "/>
xsd/NeTEx_publication_timetable.xsd-6383-			<xsd:field xpath="@id"/>
xsd/NeTEx_publication_timetable.xsd-6384-			<xsd:field xpath="@version"/>
xsd/NeTEx_publication_timetable.xsd-6385-		</xsd:key>
xsd/NeTEx_publication_timetable.xsd-6386-		<!-- =====FareProduct============================== -->
xsd/NeTEx_publication_timetable.xsd:6387:		<!-- =====FareProduct Key ========================== -->
xsd/NeTEx_publication_timetable.xsd-6388-		<!--
xsd/NeTEx_publication_timetable.xsd-6389-		<xsd:keyref name="FareProduct_KeyRef" refer="netex:FareProduct_AnyVersionedKey">
xsd/NeTEx_publication_timetable.xsd-6390-			<xsd:selector xpath=".//netex:FareProductRef |.//netex:SupplementToFareProductRef "/>
xsd/NeTEx_publication_timetable.xsd-6391-			<xsd:field xpath="@ref"/>
xsd/NeTEx_publication_timetable.xsd-6392-			<xsd:field xpath="@version"/>
--
--
xsd/NeTEx_publication_timetable.xsd-6488-			</xsd:annotation>
xsd/NeTEx_publication_timetable.xsd-6489-			<xsd:selector xpath=".//netex:PreassignedFareProduct  "/>
xsd/NeTEx_publication_timetable.xsd-6490-			<xsd:field xpath="@id"/>
xsd/NeTEx_publication_timetable.xsd-6491-			<xsd:field xpath="@version"/>
xsd/NeTEx_publication_timetable.xsd-6492-		</xsd:unique>
xsd/NeTEx_publication_timetable.xsd:6493:		<!-- =====PreassignedFareProduct Key ========================== -->
xsd/NeTEx_publication_timetable.xsd-6494-		<xsd:keyref name="PreassignedFareProduct_KeyRef" refer="netex:PreassignedFareProduct_AnyVersionedKey">
xsd/NeTEx_publication_timetable.xsd-6495-			<xsd:selector xpath=".//netex:PreassignedFareProductRef"/>
xsd/NeTEx_publication_timetable.xsd-6496-			<xsd:field xpath="@ref"/>
xsd/NeTEx_publication_timetable.xsd-6497-			<xsd:field xpath="@version"/>
xsd/NeTEx_publication_timetable.xsd-6498-		</xsd:keyref>

Reply:
v1.1 (per 17th may 2019)

```
NeTEx_publication.xsd-6622-			</xsd:annotation>
NeTEx_publication.xsd-6623-			<xsd:selector xpath=".//netex:PreassignedFareProduct"/>
NeTEx_publication.xsd-6624-			<xsd:field xpath="@id"/>
NeTEx_publication.xsd-6625-			<xsd:field xpath="@version"/>
NeTEx_publication.xsd-6626-		</xsd:unique>
NeTEx_publication.xsd:6627:		<!-- =====PreassignedFareProduct Key ========================== -->
NeTEx_publication.xsd-6628-		<xsd:keyref name="PreassignedFareProduct_KeyRef" refer="netex:PreassignedFareProduct_AnyVersionedKey">
NeTEx_publication.xsd-6629-			<xsd:selector xpath=".//netex:PreassignedFareProductRef | .//netex:SupplementToFareProductRef"/>
NeTEx_publication.xsd-6630-			<xsd:field xpath="@ref"/>
NeTEx_publication.xsd-6631-			<xsd:field xpath="@version"/>
NeTEx_publication.xsd-6632-		</xsd:keyref>
```